.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: spicy-format
+  name: Format .spicy files
+  description: This hooks formats all .spicy files
+  entry: spicy-format -s -i
+  language: rust
```

Comment:
It is kind of ugly that this repo prepares binaries, but we require users to have Rust installed to compile the latest version, see https://github.com/pre-commit/pre-commit/issues/1453. I still have hope that they might fix that issue, so could you rename this to `spicy-format-src` for now? That would be similar in spirit to what e.g., the [typos](https://github.com/crate-ci/typos/blob/master/.pre-commit-hooks.yaml) linter does (in addition to `typos-src` they also provide `typos` which wraps their binaries in a Python project, but no way to go there here yet).

Reply:
Oh... does the pre-commit action come or install rust?

Reply:
Okay, I was mistaken in thinking this required a system Rust installation. When `pre-commit` tries to install a `rust` language hook it actually installs a Rust toolchain into its local cache before compiling the binary which is both amazing and ludicrous.

With that in mind I am fine with this hook just called `spicy` since apart from potentially taking a longer time to initialize it "feels" like a binary install to the user.